ui/src/style/variables.less

```diff
@@ -73,6 +73,11 @@
 @drawer-paper-clipping-width: @paper-summary-width + (@paper-padding * 2);
 @drawer-paper-clipping-height: 200px;
 
+/**
+* Z INDEXS
+*/
+@drawer-close-button-index: 10;
```

Comment:
```suggestion
@drawer-close-button-z-index: 10;
```

Reply:
Also, why 10? Can you set using one of the values from the Material UI guidelines? https://material-ui.com/customization/z-index/

Reply:
Yup, setting it to the 'drawer' index (1200)